tests/test_scheduler.py

```diff
@@ -287,6 +287,11 @@ def check_over_configs(self, time_step=0, **config):
             elif num_inference_steps is not None and not hasattr(scheduler, "set_timesteps"):
                 kwargs["num_inference_steps"] = num_inference_steps
 
+            # Make sure `scale_model_input` is invoked to prevent a warning
+            if scheduler_class != VQDiffusionScheduler:
+                _ = scheduler.scale_model_input(sample, 0)
+                _ = new_scheduler.scale_model_input(sample, 0)
+
```

Comment:
This is not part of the UniPC failures, we can open another PR if preferred (it just removes warnings)